src/js/ApiGetter/ApiGetter.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import '../../scss/main.scss';
+
+const API_KEY = 'LR7eqPMCqaBKMlIEKf17ExpE4DlGwExDHLCXhGrZ';
+
+const TEST_ROVER = 'curiosity';
+const TEST_SOL = 1000;
+const TEST_CAMERA = 'fhaz';
+const TEST_PAGE = 1;
+
+class ApiGetter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      table: [],
+    };
+  }
+
+  getInfoFromAPI = async (rover = TEST_ROVER, sol = TEST_SOL, camera = TEST_CAMERA, page = TEST_PAGE) => {
+    const tableInfo = [];
+    const url = `https://api.nasa.gov/mars-photos/api/v1/rovers/${rover}/photos?sol=${sol}&camera=${camera}&page=${page}&api_key=${API_KEY}`;
+    const response = await fetch(url);
+    const responseJson = await response.json();
+    await responseJson.photos.forEach(elem => {
```

Comment:
I think we should be strongly assured that everything goes its way. Wrap each api request with try-catch block. It's extremely important